templates/santa/run.html.twig

```diff
@@ -328,6 +342,23 @@
                     return userList.querySelectorAll(selector+' input[type="checkbox"]');
                 }
 
+                /**
+                 * Check all user in a group
+                 */
+                function checkAllUserInAGroup(e) {
+                  const userIds = e.target.getAttribute('data-user-ids').split('|');
+
+                  for(var i=0; i<userIds.length; i++) {
```

Comment:
`const` instead of `var` ?

Reply:
`let` because it will be incremented

Reply:
Apart for `fetch` usage introduced a week ago for an optional usage, we did not use yet any "modern" JS syntax because JS is inlined in the HTML and we don't want (and don't need) to use a build tool for the frontend.
Hovewer, given the [support of `const`](https://caniuse.com/#feat=const) and [`let`](https://caniuse.com/#search=let), I guess we could start to use them.
But anyway, this should be done on another PR :wink:

Reply:
you are using `const` 2 lines above :joy:

Reply:
Indeed :stuck_out_tongue: I will change it then :+1: